usaspending_api/awards/models.py

```diff
@@ -824,10 +824,12 @@ class TransactionFABS(models.Model):
     ultimate_parent_legal_enti = models.TextField(blank=True, null=True)
     ultimate_parent_unique_ide = models.TextField(blank=True, null=True)
     uri = models.TextField(blank=True, null=True, db_index=True)
+    legal_entity_foreign_descr = models.TextField(blank=True, null=True)
```

Comment:
Please move between the fields they are between in the DB (`legal_entity_city_code` and `place_of_perfor_state_code`)

Reply:
I'm a bit confused by this one.  Pretty much none of the columns line up between the DBs I'm looking at and the Django model.  Should I rearrange all of the columns so they match?

Reply:
I wouldn't rearrange all of them. But when you are adding new columns you might as well order them to match the DB.

My guess is I can predict the table ordering by looking at the script used to reload the data.

Reply:
Well, I moved it to beneath `legal_entity_city_code` as it is in the database.  Just know that `legal_entity_city_code` in the database is nowhere near where it is in the Django model.